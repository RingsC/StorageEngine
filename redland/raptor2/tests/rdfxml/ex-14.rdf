<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:gen="http://www.mygenealogy.com/"
         xmlns:gdc="http://www.gedcom.com/">

  <!-- Check this doesn't crash / leak memory handling propElm rdf:resource -->

  <!-- Can't remember where this example came from, I've anonymised it -->

  <rdf:Description rdf:about="http://example.org/person1/">
    <gen:Parent rdf:resource="#Person_B"/>
  </rdf:Description>
 
  <rdf:Description rdf:ID="Person_B">
    <gdc:Name>John Doe</gdc:Name>
    <gdc:Birthplace>New York</gdc:Birthplace>
    <gdc:DOB>19010101</gdc:DOB>
  </rdf:Description>
</rdf:RDF>
