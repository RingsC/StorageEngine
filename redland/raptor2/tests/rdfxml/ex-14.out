<http://example.org/person1/> <http://www.mygenealogy.com/Parent> <http://librdf.org/raptor/tests/ex-14.rdf#Person_B> .
<http://librdf.org/raptor/tests/ex-14.rdf#Person_B> <http://www.gedcom.com/Name> "John Doe" .
<http://librdf.org/raptor/tests/ex-14.rdf#Person_B> <http://www.gedcom.com/Birthplace> "New York" .
<http://librdf.org/raptor/tests/ex-14.rdf#Person_B> <http://www.gedcom.com/DOB> "19010101" .
